libevm/VM.h

```diff
@@ -97,19 +101,17 @@ class VM: public VMFace
 	// space for memory
 	bytes m_mem;
 
-	// space for code and pointer to data
-	bytes m_codeSpace;
-	byte* m_code = nullptr;
+	// space for code
+	bytes m_code;
 
-	// space for stack and pointer to data
-	u256 m_stackSpace[1025];
-	u256* m_stack = m_stackSpace + 1;
-	ptrdiff_t stackSize() { return m_SP - m_stack; }
+	// space for data stack, grows towards smaller addresses from the end
+	u256 m_stack[1024];
```

Comment:
I'm not sure it will be very useful here, but you can use `std::array<u256, 1024>` instead of `u256[1024]`. It is a wrapper for a classic array, but with interface similar to other containers. So you will get a methods like `.begin()`. On the other hand, `std::begin()` function works with `u256[1024]` type.